python/pyspark/mllib/clustering.py

```diff
@@ -96,6 +96,9 @@ def k(self):
 
     def predict(self, x):
         """Find the cluster to which x belongs in this model."""
+        if isinstance(x, RDD):
+            return x.map(lambda v: self.predict(v))
```

Comment:
I caught the same thing here, https://github.com/apache/spark/pull/6499/files#diff-21c55b407050d37f67a2919470e047ebR104
It can be written as just `x.map(self.predict)`